src/scale/helper.ts

```diff
@@ -17,24 +17,20 @@
 * under the License.
 */
 
-import { NumericAxisBaseOptionCommon } from '../coord/axisCommonTypes';
-import {getPrecisionSafe, getPrecision, round, nice, quantityExponent} from '../util/number';
+import {getPrecision, round, nice, quantityExponent} from '../util/number';
 import IntervalScale from './Interval';
-import { getScaleExtent } from '../coord/axisHelper';
-import { AxisBaseModel } from '../coord/AxisBaseModel';
 import LogScale from './Log';
 import Scale from './Scale';
-import { warn } from '../util/log';
 
-const mathLog = Math.log;
+export const mathLog = Math.log;
 
```

Comment:
I ever thought if it may be slightly better to move all the mathematic functions to a common helper. Any idea? If it's feasible, that'll be a huge job.

Reply:
Removed this export.

> I ever thought if it may be slightly better to extract all the mathematic functions to a common helper

I'm not sure about it. It will be helpful if some mathematic functions needs to be polyfilled(e.g. Math.hypot) or implentated. But If we only export the methods that exits in the `Math` module. It seems to be too tricky to reduce the code size.

Reply:
@plainheart Seems d3 did the similar thing:

https://github.com/d3/d3-geo/blob/main/src/math.js#L21

Reply:
It should reduce some redundant `Math`, `degree to radian`, and other commonly used values like `2π, 1/2π` to shrink the bundle size. But this may be insignificant.